create_cluster.py

```diff
@@ -563,11 +564,12 @@ def print_failure_message():
 @click.option('--docker-image', help='Docker image to use (default: use latest planb-cassandra)')
 @click.option('--sns-topic', help='SNS topic name to send Auto-Recovery notifications to')
 @click.option('--sns-email', help='Email address to subscribe to Auto-Recovery SNS topic')
+@click.option('--application-name', help='Please specify the application name to be used')
```

Comment:
maybe rename to "appdynamics-application"? "application-name" alone is confusing (conflicts with Kio application ID)